mystery_o_matic/output/latex/__init__.py

```
@@ -0,0 +1,148 @@
+        args["introLocation"] = introLocation
+        args["initialClues"] = initial_clues
+        args["additionalClues"] = additional_clues_enumeration
+        tables = ""
+
+        for i, x in enumerate(locations.weapon_locations.items()):
+            tables += generate_latex_clue_table(f"ROOM{i}REP", len(intervals), len(mystery.get_characters()), i == 0) + "\n"
+
+        args["cluesTables"] = tables
+        args["weaponsTable"] = generate_latex_weapons_table(len(weapons_options))
+
+        for (i, char) in enumerate(mystery.get_characters()):
```

Ruff

Loop control variable `char` not used within loop body

(B007)